INSTALL

```diff
@@ -32,12 +50,6 @@ SDL_Net (optional)
     For modem/ipx support. Get it from http://www.libsdl.org/projects/SDL_net/
     Licensed under LGPL
 
-SDL_Sound 
-    For compressed audio on diskimages. (optional)
-    This is for cue/bin cdrom images with compressed (mp3/ogg) audio tracks.
-    Get it from http://icculus.org/SDL_sound
-    Licenced under LGPL
-
 ALSA_Headers
     (optional)
     for Alsa support under linux. Part of the linux kernel sources
```

Comment:
Turns out GitHub interface does not allow reviewer to attach comments to lines outside of diff context…

About lines 58, 59:

    If you want compile from developer sources (SVN) under a unix system, you'll also need 
    automake (>=1.6), autoconf(>=2.50). Should be available at http://www.gnu.org

Mention `autoconf-archive` here, please. I did it originally in a patch, that was sent to SVN upstream in previous series (5722a4e4fe), but this change is "in limbo" - neither accepted by Qbix nor refused.

Reply:
Fixed in 9bff690fd07b4bd975289795a850a64af4979fda